shopfloor/services/checkout.py

```diff
@@ -628,8 +630,12 @@ def _select_line_move_line(self, picking, selection_lines, move_line):
             return self._select_lines_from_package(
                 picking, selection_lines, move_line.package_id
             )
-        self._select_lines(move_line)
-        return self._response_for_select_package(picking, move_line)
+
+        related_lines = selection_lines.filtered(
+            lambda l: not l.package_id and l.product_id != move_line.product_id
```

Comment:
@jbaudoux the change is still here or... I'm drunk :smile:  :thinking:

Reply:
But we don't make it here when we're scanning a package, this is part of `_select_line_move_line`. In fact, this will take us to the `select_package` screen, right?

Reply:
see the condition just above, if the line is in a package, it jumps to _select_lines_from_package

So this change here is correct

Reply:
Ok, but why do you want to show lines that have nothing to deal w/ the current line?
When you select a specific line that has no package you should not list any other line IMO.
But again... I might miss the ctx or the final goal.

Reply:
How can you pack the products without seeing the lines? After selecting the first line, I need to see the other lines otherwise I cannot select more than one line to put in a pack